app/src/main/java/com/launchdarkly/hello_android/MainActivity.java

```diff
@@ -9,30 +9,30 @@
 import com.launchdarkly.sdk.android.LDClient;
 import com.launchdarkly.sdk.android.LDConfig;
 
+// Set FLAG_KEY to the feature flag key you want to evaluate.
 public class MainActivity extends AppCompatActivity {
     private static final String TAG = "LDMainActivity";
-    private static final String FLAG_KEY = "myBooleanFlag";
+    private static final String FLAG_KEY = "my-boolean-flag";
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+// Set mobileKey to your LaunchDarkly mobile key.
         LDConfig ldConfig = new LDConfig.Builder()
                 .mobileKey("MOBILE_KEY")
                 .build();
 
-        LDUser user = new LDUser.Builder("user key")
-                .email("fake@example.com")
+// Set up the user properties. This user should appear on your LaunchDarkly users dashboard soon after you run the demo.
+        LDUser user = new LDUser.Builder("example-user-key")
+                .name("Sandy")
                 .build();
 
         LDClient client = LDClient.init(getApplication(), ldConfig, user, 5);
 
-        if (client.boolVariation(FLAG_KEY, false)) {
-            Log.i(TAG, "Showing feature");
-        } else {
-            Log.i(TAG, "Not Showing feature");
-        }
+        boolean flagValue = client.boolVariation(FLAG_KEY, false)
+                Log.i(TAG, "Feature flag '" + FLAG_KEY + "' is " + flagValue + " for this user");
```

Comment:
Saw your earlier comment...I'm not sure if this syntax is correct either. It's possibly 
```
Log.i(TAG, "Feature flag [%s] is [%s] for this user", FLAG_KEY, flagValue)
```
based on a quick search in the [sdk repo](https://github.com/launchdarkly/android-client-sdk/search?q=log). Not sure though.